docs/CONTRIBUTING.md

```diff
@@ -293,7 +293,7 @@ This will install a binary named `diesel`, so you should be able to run `diesel
 --version` to confirm successful installation.
 
 > If you're on Linux and this fails with an error that looks like `error:
-linking with `cc` failed: exit code: 1`, you're probably missing some
+linking with`cc`failed: exit code: 1`, you're probably missing some
```

Comment:
why did you remove the spaces?

Reply:
Good catch. I think the linter thought it was `a` and `b`, but it actually is `a and b`. I will fix it.

Reply:
Updated.

Reply:
you can escape backticks by doubling the number of outer backticks.

for example: 

```
``foo`bar`baz``
```

will render as ``foo`bar`baz``

I've added a small commit on top that adjusts this :)